User interface updates.

xcguy opened the pull request:
Fix issue with acceleration tile.
Degree symbol on temperature.
Use fixed lux unit (so it displays correctly in logs, activity
messages).
Round temperature for display.
Minor cleanups in help text.

xcguy committed:
User interface updates.

Fix issue with acceleration tile.
Degree symbol on temperature.
Use fixed lux unit (so it displays correctly in logs, activity
messages).
Round temperature for display.
Minor cleanups in help text.

CyrilPeponnet commented on device-handler/FibaroMotionSensor/FibaroMotionSensor.groovy:
```diff
@@ -100,14 +100,14 @@ metadata {
-            state("still", label:'${currentValue}', icon:"st.motion.acceleration.inactive", backgroundColor:"#ffffff")
+            state("active", label:'Vibration', icon:"st.motion.acceleration.active", backgroundColor:"#53a7c0")
+            state("inactive", label:'Still', icon:"st.motion.acceleration.inactive", backgroundColor:"#ffffff")
         }
```
Actually this tile is working when you set your device to vibration mode.

xcguy replied:
Initially, my unit reported 'inactive'.   After your comment, I knocked on the sensor to trigger the event and it did indeed show numbers (acceleration), and then finally 'still'.  
I'd suggest keeping your code, but adding the one state for 'inactive' to cover that one end case.

CyrilPeponnet replied:
Make sense. Go ahead.

CyrilPeponnet commented:
Looks good except for the vibration tile.

xcguy committed:
Revised acceleration states.

Attempted to cover all possible acceleration states (reviewed code from
other Fibaro Motion Sensors).

xcguy commented:
Revised the pull, should cover all the possible states.

Nice device code, much better than the earlier attempts by others!

CyrilPeponnet commented:
Thanks I appreciate :)
